src/github.com/stellar/gateway/protocols/attachment/attachment.go

```diff
@@ -0,0 +1,55 @@
+package attachment
+
+import (
+	"encoding/json"
+)
+
+// Attachment represents core object in Stellar attachment convention
+type Attachment struct {
```

Comment:
Perhaps this should be renamed to `Envelope`.   We should avoid stutter when we can. https://blog.golang.org/package-names has some discussion on this.